src/components/preview/preview.pw.tsx

```diff
@@ -0,0 +1,147 @@
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+import { dlsRoot } from "../../../playwright/components/index";
+import {
+  assertCssValueIsApproximately,
+  checkAccessibility,
+} from "../../../playwright/support/helper";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const pixelsData = [256, 275, 300];
+
+test.describe("check Preview component properties", () => {
+  [...pixelsData].forEach((height) => {
+    test(`should check height as ${height}px for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PreviewComponent height={`${height}px`} />);
+
+      await assertCssValueIsApproximately(
+        previewComponent(page),
+        "height",
+        height
+      );
+    });
+  });
+
+  [...pixelsData].forEach((width) => {
+    test(`should check width as ${width}px for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PreviewComponent width={`${width}px`} />);
+
+      await assertCssValueIsApproximately(
+        previewComponent(page),
+        "width",
+        width
+      );
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children} for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<Preview>{children}</Preview>);
+
+      await expect(dlsRoot(page)).toHaveText(children);
+    });
+  });
+
+  test("should check when loading is True for Preview component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<PreviewComponent loading />);
+
+    await expect(previewComponent(page)).toBeVisible();
+  });
+
+  test("should check when loading is False for Preview component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<PreviewComponent loading={false} />);
+
+    await expect(previewComponent(page)).not.toBeVisible();
+  });
+
+  [5, 6, 8, 10].forEach((line) => {
+    test(`should check ${line} loading lines for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PreviewComponent lines={line} />);
+
+      const elementsCount = await lineComponent(page).count();
+      await expect(elementsCount).toBe(line);
+    });
+  });
+});
+
+test.describe("Accessibility tests for Preview component", () => {
+  [...pixelsData].forEach((height) => {
+    test(`should check accessibility when height as ${height}px for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PreviewComponent height={`${height}px`} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  [...pixelsData].forEach((width) => {
+    test(`should check check accessibility when width as ${width}px for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PreviewComponent width={`${width}px`} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  [testData].forEach((children) => {
```

Comment:
**suggestion**: `testData` doesn't need to be placed in square brackets here. You can just leave it as `testData.forEach...` to match the others